src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js

```diff
@@ -64,13 +64,18 @@ const determineIfProtectionOrder = (documents) => {
 };
 
 export default function PackageReview() {
+  const rushTabFeatureFlag = window.env
+    ? window.env.REACT_APP_RUSH_TAB_FEATURE_FLAG
+    : process.env.REACT_APP_RUSH_TAB_FEATURE_FLAG;
```

Comment:
Would be nice if this went in EnvConfig.js to consolidate all env vars.

Reply:
Good point, I didn't look at the EnvConfig.js file when I created this. This is an old PR but definitely something we should update in the future.